support/crown/src/common.rs

```diff
@@ -276,11 +238,43 @@ pub fn def_path_res(cx: &LateContext<'_>, path: &[&str]) -> Vec<Res> {
     resolutions
 }
 
-/// Resolves a def path like `std::vec::Vec` to its [`DefId`]s, see [`def_path_res`].
-pub fn def_path_def_ids(cx: &LateContext<'_>, path: &[&str]) -> impl Iterator<Item = DefId> {
-    def_path_res(cx, path)
+pub fn def_local_res(cx: &LateContext<'_>, path: &str) -> Vec<Res> {
```

Comment:
The rustdoc here used to explain what this function does, but now it's just called `def_local_res` so I don't really know. What does it to and can that description be translated into rustdoc?

Reply:
I will improve docs.